docs/src/components/Layout/Sidebar.tsx

```diff
@@ -141,9 +141,12 @@ const SecondaryNav = (props) => {
   const isReactNative = platform === 'react-native';
   const isAndroid = platform === 'android';
   const isSwift = platform === 'swift';
+  const isAngular = platform === 'angular';
+  const isVue = platform === 'vue';
 
   const hideTheming = isAndroid || isSwift;
-  const hideGuidesExpander = isFlutter || isReactNative || isAndroid || isSwift;
+  const hideGuidesExpander =
+    isFlutter || isReactNative || isAndroid || isSwift || isAngular || isVue;
```

Comment:
Since `CSS in JS` was the only guide for Vue and Angular, I'm hiding the section now since this page shouldn't be shown for these platforms